incubator/hnc/pkg/controllers/hierarchy_controller.go

```diff
@@ -150,6 +151,10 @@ func (r *HierarchyReconciler) reconcile(ctx context.Context, log logr.Logger, nm
 // fully written back to the apiserver yet, each namespace is reconciled in isolation (apart from
 // the in-memory forest) so this is fine.
 func (r *HierarchyReconciler) syncWithForest(log logr.Logger, nsInst *corev1.Namespace, inst *api.HierarchyConfiguration) {
+	if !ex[inst.ObjectMeta.Namespace] {
```

Comment:
Why are we only excluding the _metrics_ if the namespace is excluded? Shouldn't this just be:

```
if ex[inst.ObjectMeta.Namespace] {
  return
}
```

And similarly elsewhere in this file?